examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -12,9 +14,12 @@
 from torch.optim.lr_scheduler import StepLR
 from torchvision import datasets, transforms
 
+# Docs snippet start after: import determined
```

Comment:
You don't actually have this as a literal include.

Reply:
removed the comment